src/transformers/pipelines/__init__.py

```diff
@@ -305,14 +305,14 @@
     elif values["type"] != "multimodal":
         raise ValueError(f"SUPPORTED_TASK {task} contains invalid type {values['type']}")
 
+PipelineRegistry = _PipelineRegistry.from_dict(supported_tasks=SUPPORTED_TASKS, task_aliases=TASK_ALIASES)
```

Comment:
NIT: As a global and not a class name I would suggest using `PIPELINE_REGISTRY` instead.
Having a registry as a global variable is perfectly fine to me.

Also `from_dict` could be just a regular `__init__` maybe a bit more pythonic ?

Reply:
I don't have a big preference on this. I will update accordingly.

Reply:
Done.